src/server/plugins.js

```diff
@@ -3,13 +3,19 @@
 var fs = require('fs'),
     path = require('path'),
     cordovaServe = require('cordova-serve'),
-    pluginMapper  = require('cordova-registry-mapper').oldToNew,
+    pluginMapper = require('cordova-registry-mapper').oldToNew,
     config = require('./config'),
-    dirs = require('./dirs');
+    dirs = require('./dirs'),
+    telemetry = require('./telemetry-helper');
 
 var pluginSimulationFiles = require('./plugin-files');
 
 var plugins = {};
+var pluginsTelemetry = {
+    simulatedBuiltIn: [],
+    simulatedNonBuiltIn: [],
+    notSimulated: []
+};
```

Comment:
I would extract this init code (for `plugins` and `pluginsTelemetry` variables) to a function and reuse it in `initPlugins()`. It feels like we are initializing there again.